### ISSUE TITLE
Lasse gave up - time to change...

### ISSUE THREAD

**Issue by NilsHoyer** [bonecode/TBCEditor#1]
Lasse, the orignal developer of the TBCEditor, gave up this project. He deleted this repository and the bonecode account at GitHub completely.

Some days before, I received this note from Lasse:

>   There is about 40 days time to finish this. After that I am going to travel around the world (3rd time) and after that I will finish my 3D engine which may still take a year or two...

The mail address I used in the last days to communicate with him does not longer work.

Yesterday, I wrote a E-Mail to him ... the mail address I contacted him some days before is not longer valid.

Further informations I don't have. May he find, what he is looking for...

Now it's time to change. I re-created the bonecode user account an filled the repositories with the last updates, I got. Now I will endeavor to provide this project further more and I hope to get a from assistance from other people too.

**Comment by delphioracle** [comment]
> I re-created the bonecode user account an filled the repositories with the last updates, I got. 

Thank you very much.

Now, I suggest to create JiftCode user account and to fork all repositories.
Make BoneCode ------> JiftCode links.

**Comment by Memnarch** [comment]
Thanks for recreating. Which is the last update? You didn't push the old history(shame on you :P).

**Comment by bonecode** [comment]
@Memnarch 
The code is from 09 Feb 2017 - I don't have the old history (the old repository). I asked GitHub about it, but they wont restore it.

@delphioracle 
Why not work with this bonecode account further more? Do you have panic, I will do the same like Lasse?

Please give me some days to understand and handle the BCEditor ... I never worked with it before. Than I hope to offer support and updates like Lasse did.

**Comment by Memnarch** [comment]
@bonecode you didn't maintain a local git-mirror oO?
Maybe i can ask someone to give you his history. His one is from 9th February, too

**Comment by NilsHoyer** [comment]
@Memnarch
The code in the repository is from 09 Feb 2017 - I don't have the old history (the old repository). I asked GitHub about it, but they wont restore it.

@delphioracle
Why not work with this bonecode account further more? Do you have panic, I will do the same like Lasse?

Please give me some days to understand and handle the BCEditor ... I never worked with it before. Than I hope to offer support and updates like Lasse did.

**Comment by delphioracle** [comment]
> Why not work with this bonecode account further more? 

1) We would like to thank  Lasse Rautiainen for his great efforts, and of course all the other developers of the BoneCode project. But the mysterious final and karma of the project can negatively affect his successful continuation. Remember Thomas Jaeger yesterday's comments.
2) We must save the original for history and as reference point.

**Comment by maciej-izak** [comment]
https://github.com/maciej-izak/TBCEditor/commits/master

I have almost full history... Last commit from Feb 5, 2017 (1928 commits)

**Comment by NilsHoyer** [comment]
... so I should delete this repository again and fork it from yours? Than I commit the changes up to 09 Feb 2017. If I'm right, on this way we have the most history of Lasses work at the old address.

Should I try that?

**Comment by quartexNOR** [comment]
Sure that is kosher? From what I heard Lasse went closed repo due to some internal conflicts between you nils and himself.
Doubt people will touch this code until Lasse gives the OK for you to just re-create his repo and continue. From what i see there is code from synedit mixed with a code that isnt even marked.
Please talk to lasse and have him issue a public statement.

**Comment by maciej-izak** [comment]
Do what you need :) I am a little surprised that old repo/organization was deleted... Very strange. Feel free to fork and please make update.

**Comment by ThomasJaeger** [comment]
@NilsHoyer Why did you delete all of my previous comments?

**Comment by NilsHoyer** [comment]
@ThomasJaeger 
I deleted this repository and recreated it again.

**Comment by NilsHoyer** [comment]
@quartexNOR 
You are right: I told Lasse my opinion about this kind of thinking and handling my suggestions. From my point of view, he was selfish, childish and arrogant. And if I think this from somebody, I tell this to him. That's all, I done.

After he deleted this repository and gave up this project completely, I feel to be confirmed about my opinion of him.

It was never my planning to overtake this project. But I feel to be able to do this - with support and updates. Maybe it's god planning, right now I have to do this...

What should I do now? Should I go away? Should I delete this account again?

PS
Maybe I delete this repository again to fork that other repository. Than your new comments will be deleted again - I'm sorry about this...

**Comment by maciej-izak** [comment]
IMO we should to keep this repository. New owner of some open source project is natural thing. IMO deleting this root repo by original author was very selfish...

IMO the most important is to keep history of project in this place.

**Comment by maciej-izak** [comment]
So you can delete it and bring it back with full history :) nothing bad with that.

**Comment by Memnarch** [comment]
sglienke has a copy of 9th February ;). That one seems to be out of date by 3 days.
